Gemfile

```diff
@@ -13,3 +13,5 @@ gemspec
 # To use a debugger
 # gem 'byebug', group: [:development, :test]
 
+
+gem 'rspec-html-matchers', git: 'https://github.com/ministryofjustice/rspec-html-matchers.git', branch: 'fix-deep-nesting'
```

Comment:
The gem itself is already in the `gemspec`, it was merged in with `cleanup-error-helpers-spec`. I don't believe we can specify a git repo or path with `add_development_dependency`, so this is in the Gemfile just for that purpose.